docs/examples/floatingpoint.rst

```diff
@@ -4,4 +4,5 @@ Theory of Floating-Points
 
 .. api-examples::
     <examples>/api/java/FloatingPointArith.java
+    <z3pycompat>/test/pgms/example_floatingpoint.py
```

Comment:
There is something wrong here...

Reply:
Good catch. Yes. It should be fixed in c60228f